pkg/api/routes.go

```diff
@@ -1052,6 +1052,10 @@ func (rh *RouteHandler) DeleteBlob(response http.ResponseWriter, request *http.R
 			zcommon.WriteJSON(response,
 				http.StatusNotFound,
 				apiErr.NewErrorList(apiErr.NewError(apiErr.BLOB_UNKNOWN, map[string]string{".String()": digest.String()})))
+		} else if errors.Is(err, zerr.ErrBlobBusy) {
+			zcommon.WriteJSON(response,
+				http.StatusConflict,
+				apiErr.NewErrorList(apiErr.NewError(apiErr.BLOB_UNKNOWN, map[string]string{".String()": digest.String()})))
```

Comment:
Add a new error under pkg/api/errors?

Reply:
can't do it, api errors should be compliant with dist-spec.

Reply:
I think we should use 405 Method Not Allowed, in the spec they mention 405 as a valid response for failure to delete.
I think we should use the `DENIED` from the dist-spec.

```

```